### PR TITLE
added tests for twitter, changed request.

### PR DESCRIPTION
**Pull request by decobots**
I will remake Twitter class to use request in next task.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 *.jpg
 *.jpeg
 .cache
+.pytest_cashe
```
Comment (livercat) [inline .gitignore]: Typo: cache

File: test_twitter [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import pytest
```
Comment (livercat) [inline test_twitter]: What is this file?

File: test_twitter.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    assert isinstance(returned_name, str)
+    assert returned_id != ''
+    assert returned_name != ''
+
+
+def test_upload_photo_incorrect(local_picture):
+    twitter = Twitter()
+    not_picture = "string"
+    with pytest.raises(TypeError):
+        twitter.upload_photo(name="Test_name", data=not_picture)
```
Comment (livercat) [inline test_twitter.py]: Only this line will execute, then exception will be raised and the next line will be skipped

File: test_twitter.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        twitter.upload_photo(name=78.6765, data=local_picture)
+
+
+def test_create_post_correct(picture_id):
+    twitter = Twitter()
+    twitter.create_post(status="Test_status", id_of_photo=picture_id)
```
Comment (livercat) [inline test_twitter.py]: Check for id?

File: .gitignore (hunk 1 above)
Comment (decobots) [reply .gitignore]: not in my English))) fixed

File: test_twitter.py (hunk 3 above)
Comment (decobots) [reply test_twitter.py]: fixed

File: test_twitter (hunk 2 above)
Comment (decobots) [reply test_twitter]: will be deleted

File: test_twitter.py (hunk 4 above)
Comment (decobots) [reply test_twitter.py]: it will stay as it is

**Pull request merged**